Makefile

```diff
@@ -31,7 +34,7 @@ db/rollback:
 	bee migrate rollback -driver=postgres -conn="$(DATABASE_URL)"
 
 install/package:
-	npm i
+	source $(HOME)/.nvm/nvm.sh && nvm use && npm i
```

Comment:
By adding a file `.nvmrc`, this version can be set for the project. Also adding `source $(HOME)/.nvm/nvm.sh` is more of a personal setting in your bashrc or zhrc. I would not expect this in the project file.